### PR TITLE
OCPBUGS#15423: Fixed link pointing to RHV-H for vSphere docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-15423](https://issues.redhat.com/browse/OCPBUGS-15423)

Version(s):
4.14 through to 4.10

Link to docs preview:
[VMware vSphere CSI Driver Operator requirements](https://61947--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/preparing-to-install-on-vsphere.html#vsphere-csi-driver-reqs_preparing-to-install-on-vsphere)

QE review:
- [x] QE has approved this change.

Additional information:
[Request to RHEL writers in Slack](https://redhat-internal.slack.com/archives/C059QKXFLTS/p1688134715743949)

### CODE REVIEW

**Comment by dfitzmau** [comment]
Hi @sgaoshang . Would you be OK to review this PR?

File: modules/installation-vsphere-infrastructure.adoc
Comment (skopacz1) [inline modules/installation-vsphere-infrastructure.adoc:38]: I'd suggest removing italic formatting from "Red Hat Customer Portal". Although I can't find a style guide reference that says you can't, none of the references to the portal I found in the docs have italics.

**Review by skopacz1 — COMMENTED** [review]
Looks good! I just left one suggestion. I'm new to reviewing, so please wait until @adellape approves this review before implementing feedback. Thanks!

**Review by adellape — COMMENTED** [review]
@skopacz1 Peer review LGTM 👍

File: modules/installation-vsphere-infrastructure.adoc
Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc:38]: Thanks, @skopacz1 and @adellape . I struggled to find guidance also but I used the format I used to adopt in the Runtimes project space. I did see this entry in our `doc_guidelines.adoc` file if this adds more clarity to my use of italics: 

Screenshot from 2023-07-12 09-32-00

Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc:38]: WDYT? Keep italic format or remove it?

Comment (skopacz1) [reply modules/installation-vsphere-infrastructure.adoc:38]: I think this situation is a bit weird because we're not referencing any explicit set of documentation (which I believe our style guides provide guidance for), we're talking about a site where you can access a host of things.

Personally I'd still be in favor of removing the italic formatting. Alex mentioned that if there's no explicit guidance, this Support page would be a good model to follow: [Support overview](https://docs.openshift.com/container-platform/4.13/support/index.html#:~:text=Red%20Hat%20Customer%20Portal)

Ultimately it's up to you, but I think keeping it plain format would keep it more consistent with other instances in our documentation. Either way this is likely something worth clarifying in some style or contribution guide.

**Review by jeana-redhat — APPROVED** [review]
LGTM
/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by jeana-redhat** [comment]
@dfitzmau you'll need to do the CP into 4.10&4.11 manually unfortunately

**Pull request merged**
